staging/src/k8s.io/cri-api/pkg/apis/runtime/v1alpha2/api.proto

```diff
@@ -949,6 +949,8 @@ message ContainerStatus {
     repeated Mount mounts = 14;
     // Log path of container.
     string log_path = 15;
+    // Resource configuration of the container.
+    LinuxContainerResources resources = 16;
```

Comment:
Does this require changes from the VM runtime server(virtlet)?

Reply:
It is not required, but would be ideal if runtime reported this. Currently, I'm handling the case where runtime does not support returning this info by assuming initially configured resource limits, or the limits from last successful UpdateContainerResources API call. If runtime reports this info, the runtime report gets precedence.